src/test/java/spoon/test/logging/LogTest.java

```diff
@@ -87,5 +91,24 @@ public void testMavenLauncherLogs() {
 			mavenLauncher = new MavenLauncher("./pom.xml", MavenLauncher.SOURCE_TYPE.APP_SOURCE, new String[]{"./"});
 			assertEquals("Running in FULLCLASSPATH mode. Classpath is manually set (doc: http://spoon.gforge.inria.fr/launcher.html).",logger.getLoggingEvents().get(0).getMessage());
 		}
+
+		@Test
+		public void testLoggingOff() {
+			final TestLogger logger = TestLoggerFactory.getTestLogger(Launcher.class);
```

Comment:
Could you add a contract as a comment at the top of the test method? I.e. a comment stating what the test should do along, like so (ofc replacing my nonsense text):

```suggestion
		@Test
		public void testLoggingOff() {
			// contract: The logger should do X given Y and perhaps Z
			final TestLogger logger = TestLoggerFactory.getTestLogger(Launcher.class);
```